src/classes/frDonation.cls

```diff
@@ -41,8 +41,10 @@
 public class frDonation extends frModel implements frSyncable {
     public static List<frMapping__c> mappings {
         get {
+            String objType = TYPE;
+            if(frUtil.hasNPCobjects()) objType = 'Gift Transaction';
```

Comment:
Since this is changing the TYPE variable that is used in a few different places, is it possible to refactor the way TYPE is defined or called so we don't have to do this type of checking each time we access TYPE?   Right now the frSetupController is using the TYPE value defined in this class, but it is being similarly overwritten with different logic there.  

Would it be possible to have it such that there is a single frDonation.TYPE value and that resolves to the correct value based on the environment detected, so both the class that sets up the mappings and the class that uses the mapping (here) are referencing the same value?  That way we can be certain the queries are using the same criteria and it is easier to refactor if it needs to be changed in the future

Reply:
Added this in the new update! Just checks if its an org and assigns the type variable accordingly.